### PR TITLE
fix wrong version of, and missing, extensions for Perl, detected by easyblock PR 2699

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

Depends on (or rather fixes problems detected by) https://github.com/easybuilders/easybuild-easyblocks/pull/2699

### CODE REVIEW

**Comment by akesandgren** [comment]
@boegel Can you kick of a test of these with the 2699 easyblock PR.
I think I got them all right now...

**Comment by branfosj** [comment]
@boegelbot please test @ generoso
EB_ARGS="--include-easyblocks-from-pr 2699"

**Comment by branfosj** [comment]
Test report by @branfosj
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2699
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/branfosj/13960708c8bfb28aec07a24bd8d4f4b2 for a full test report.

File: easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -1761,11 +1763,6 @@ exts_list = [
         'source_urls': ['https://cpan.metacpan.org/authors/id/M/ML/MLEHMANN/'],
         'checksums': ['a86a1c4ca4f3006d7479064425a09fa5b6689e57261fcb994fe67d061cba0e7e'],
     }),
-    ('IO::Compress::Bzip2', '2.096', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb:1764]: @akesandgren Can you clarify why this has to be removed?

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2
EB_ARGS="--include-easyblocks-from-pr 2699"

File: easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb:1764]: @akesandgren ping?

Comment (akesandgren) [reply easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb:1764]: It's part of Perl main code if I remember correctly now....

Comment (akesandgren) [reply easybuild/easyconfigs/p/Perl/Perl-5.30.2-GCCcore-9.3.0.eb:1764]: And if left something failed in the tests due to this, probably something about incompatible bzip/zip/... levels

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/1effd3d5e3ae03f410c709270cf0a847 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
Using easyblocks from PR(s) https://github.com/easybuilders/easybuild-easyblocks/pull/2699
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/489157077df4503763f421a7cb55c684 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**